### DISCUSSION TITLE
Request rejection in WalletConnect Cloud

### DISCUSSION THREAD

**Discussion by damianlluch** [WalletConnect/walletconnect-monorepo#4237] [Developer Support]
Hi, I'm in the super difficult task of migrating a project with WalletConnectV1 to V2.

Question, already in the programming it is complex to make the change of version. And in the cloud they are rejecting my project for this reason, which I really don't know what it refers to.
I don't see any option to contact support, and I don't see any clarification on what part of the form it refers to.
I put this question here, since I don't know who to ask.

Thank you

```
Changes requested
Your update has been rejected. Reason:

"Dapp Testing - Could not verify if testing has been done on this dapp. As we do not have available V2 metric data for Sign. Please test your integration with our test wallet and submit for approval if successful. Link: https://react-web3wallet.vercel.app"```

**Comment by glitch-txs** [comment]
hi, what's the project ID?

**Reply by damianlluch** [reply]
Hi!

86dd74caf10d75e4c6be7c66bcac5ffb

**Reply by glitch-txs** [reply]
Hi, sorry for the delay, this is still being discussed but most probably we won't be accepting dapp submissions moving forward. Feel free to send an email to partnerships@walletconnect.com for any partnership related discussion. Thanks!

**Reply by damianlluch** [reply]
Hi, I didn't understand your answer.

**Reply by damianlluch** [reply]
What do you mean by "we won't be accepting dapp submissions moving forward".
Why do they ask for testing instructions for a dapp?
